src/AspNet.Security.OAuth.Moodle/AspNet.Security.OAuth.Moodle.csproj

```
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>ASP.NET Core security middleware enabling Moodle authentication.</Description>
+    <Authors>HIT-ReFreSH</Authors>
+    <PackageTags>aspnetcore;authentication;moodle;oauth;security</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="JetBrains.Annotations" Version="2020.3.0" />
```

The version is defined elsewhere, and is for development-time only.

```suggestion
    <PackageReference Include="JetBrains.Annotations" PrivateAssets="All" />
```